app/screens/card-screen/card.tsx

```diff
@@ -0,0 +1,485 @@
+        />
+      )}
+      {(!cardHtml || !refreshBalance) && (
+        <ModalNfcFlashcard
+          isActive={displayReceiveNfc}
+          setIsActive={setDisplayReceiveNfc}
+          onCardHtmlUpdate={handleCardHtmlUpdate}
+        />
+      )}
+    </Screen>
+  )
+}
+
+const useStyles = makeStyles(({ colors }) => ({
+  screenStyle: {
+    paddingHorizontal: 16,
+    paddingVertical: 12,
+    flexGrow: 1,
+  },
+
+  header: {
+    backgroundColor: colors.white,
+  },
+
+  activityIndicatorContainer: {
+    alignItems: "center",
+    flex: 1,
+    justifyContent: "center",
+  },
+
+  emptyListNoContacts: {
+    marginHorizontal: 12,
+    marginTop: 32,
+  },
+
+  emptyListScanIcon: {
+    alignItems: "center",
+    justifyContent: "center",
+    flex: 1,
+    paddingTop: 74,
+  },
+
+  emptyListNoMatching: {
+    marginHorizontal: 26,
+    marginTop: 8,
+  },
+
+  emptyListText: {
+    fontSize: 18,
+    marginTop: 30,
+    textAlign: "center",
+    color: colors.black,
+  },
+
+  emptyListTitle: {
+    color: colors.black,
+    fontSize: 24,
+    fontWeight: "bold",
+    textAlign: "center",
+  },
+
+  item: {
+    marginHorizontal: 32,
+    marginVertical: 8,
+  },
+
+  itemContainer: {
+    borderRadius: 8,
+    backgroundColor: colors.grey5,
+  },
+
+  listContainer: { flexGrow: 1 },
+
+  searchBarContainer: {
+    backgroundColor: colors.white,
+    borderBottomColor: colors.white,
+    borderTopColor: colors.white,
+    marginHorizontal: 26,
+    marginVertical: 8,
+  },
+
+  searchBarInputContainerStyle: {
+    backgroundColor: colors.grey5,
+  },
+
+  searchBarRightIconStyle: {
+    padding: 8,
+  },
+
+  searchBarText: {
+    color: colors.black,
+    textDecorationLine: "none",
+  },
+
+  itemText: { color: colors.black },
+
+  icon: {
+    color: colors.black,
+  },
+  profilePicture: {
+    width: 50,
+    height: 50,
+    borderRadius: 50,
+    backgroundColor: colors.black,
+  },
+  nfcIcon: {
+    marginTop: -1,
+    marginRight: 17,
+    padding: 8,
+    display: "flex",
+    flexDirection: "row",
+    columnGap: 4,
+    backgroundColor: colors.grey5,
+    borderRadius: 4,
+  },
+  flashcardImage: {
+    width: "100%", // Adjust the size as needed
+    height: 200, // Adjust the size as needed
+    resizeMode: "contain",
+    marginBottom: 16, // Add some space below the image
+  },
+  flashcardBalanceText: {
+    fontSize: 36,
+    marginBottom: 8,
+    marginTop: 16,
+    textAlign: "center",
+  },
+  listItemsContainer: {
+    paddingHorizontal: 15,
+    paddingVertical: 15,
+    marginBottom: 20,
+    borderRadius: 12,
+    backgroundColor: colors.grey5,
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "flex-start",
+  },
+  button: {
+    display: "flex",
+    justifyContent: "space-between",
+    width: "100%",
+    maxWidth: 184,
+  },
+  removeButton: {
+    marginTop: 16,
+    display: "flex",
+    paddingVertical: 96,
+    borderRadius: 12,
+  },
+}))
```

Comment:
Most of the styles not being used, you can remove them to make the component clean

Reply:
cleaned up all unused styles